src/js/components/DataTable/DataTable.js

```diff
@@ -413,6 +494,30 @@ const DataTable = ({
               setRowExpand={setRowExpand}
             />
           )}
+          {onMore && moreButton ? (
+            <StyledDataTableBody>
```

Comment:
Is it correct for this to introduce another TableBody inside the same table? Or should this just be added as a row in the Body?